src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,8 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        System.out.println(new Lottery().getRandomBall().toString());
+        System.out.println(new Lottery().getRandomBall().toString());
+        System.out.println(new Lottery().getRandomBall().toString());
```

Comment:
do we need 3 lotteries to generate 3 balls? :)
Also, let's use loops to create them and print info about them